eng/mgmt/automation/generation.yml

```diff
@@ -0,0 +1,61 @@
+trigger: none
```

Comment:
When do you expect someone to call this pipeline over just using the sdk automation?

Reply:
This yml is mainly used for manually generation, which will be manually run in devops.